### PR TITLE
Add double-lined class name translations

### PR DESCRIPTION
**Pull request by zenseii**
This adds translations to the double-lined class names in case there are more than 4 players in a scenario.

Relates to https://github.com/ihhub/fheroes2/issues/5248 and https://github.com/ihhub/fheroes2/issues/5244

Follow up to #5983

### CODE REVIEW

File: files/lang/ru.po [hunk 1]
```diff
@@ -6590,36 +6590,31 @@ msgid "race|Neutral"
...
 
 msgid "doubleLined|Wizard"
-msgstr ""
+msgstr "Волшебник"
 
-#, fuzzy
 msgid ""
 "doubleLined|Necro-\n"
 "mancer"
 msgstr "Некромант"
```
Comment (Districh-ru) [inline files/lang/ru.po:6610]: Hi @zenseii ,
Necromancer is the longest class name, so it should be split into two lines.
изображение

```suggestion
msgstr "Некро-\nмант"
```

Wizard also has a long class name, but we have a 2px space between its class names, so we can leave it unsplitted.

File: files/lang/ru.po [hunk 2]
```diff
@@ -6590,36 +6590,31 @@ msgid "race|Neutral"
...
 
 msgid "doubleLinedRace|Multi"
-msgstr ""
+msgstr "По сценарию"
```
Comment (Districh-ru) [inline files/lang/ru.po:6613]: изображение

```suggestion
msgstr "По\nсценарию"
```

**Comment by srzkptrsk** [comment]
> when there are 6 players.
> Thank you in advance for your help!

Something wrong with string for translate
`#, fuzzy
msgid ""
"doubleLined|Necro-\n"
"mancer"
msgstr "Некра-\n"
"мант"`

Screenshot 2022-12-03 at 16 30 23

File: files/lang/pl.po [hunk 3]
```diff
@@ -6622,36 +6622,31 @@ msgid "race|Neutral"
...
-msgstr ""
+msgstr "Rycerz"
 
-#, fuzzy
 msgid "doubleLined|Barbarian"
 msgstr "Barbarzyńca"
 
-#, fuzzy
 msgid "doubleLined|Sorceress"
 msgstr "Czarodziejka"
```
Comment (Arthusppp) [inline files/lang/pl.po:6631]: msgid "doubleLined|Sorceress"
msgstr "Czaro-\ndziejka"

File: files/lang/pl.po [hunk 4]
```diff
@@ -6622,36 +6622,31 @@ msgid "race|Neutral"
...
 
 msgid "doubleLined|Warlock"
-msgstr ""
+msgstr "Czarnoksiężnik"
```
Comment (Arthusppp) [inline files/lang/pl.po:6634]: msgid "doubleLined|Warlock"
msgstr "Czarno-\nksiężnik"

File: files/lang/pl.po [hunk 5]
```diff
@@ -6622,36 +6622,31 @@ msgid "race|Neutral"
...
 
 msgid "doubleLined|Wizard"
-msgstr ""
+msgstr "Czarodziej"
 
-#, fuzzy
 msgid ""
 "doubleLined|Necro-\n"
 "mancer"
 msgstr "Nekromanta"
```
Comment (Arthusppp) [inline files/lang/pl.po:6642]: msgid ""
"doubleLined|Necro-\n"
"mancer"
msgstr "Nekro-\nmanta"

File: files/lang/pl.po [hunk 6]
```diff
@@ -6622,36 +6622,31 @@ msgid "race|Neutral"
 msgstr "Neutralny"
 
 msgid "doubleLined|Knight"
-msgstr ""
+msgstr "Rycerz"
 
-#, fuzzy
 msgid "doubleLined|Barbarian"
 msgstr "Barbarzyńca"
```
Comment (Arthusppp) [inline files/lang/pl.po:6628]: msgid "doubleLined|Barbarian"
msgstr "Barba-\nrzyńca"

**Review by Arthusppp — CHANGES_REQUESTED** [review]

File: files/lang/bg.po [hunk 7]
```diff
@@ -6608,36 +6608,31 @@ msgid "race|Neutral"
...
 msgid "doubleLined|Barbarian"
 msgstr "Варварин"
 
-#, fuzzy
 msgid "doubleLined|Sorceress"
 msgstr "Магьосница"
 
 msgid "doubleLined|Warlock"
-msgstr ""
+msgstr "Магьосник"
```
Comment (Effektus) [inline files/lang/bg.po:6620]: Магьос- \n
ник

File: files/lang/bg.po [hunk 8]
```diff
@@ -6608,36 +6608,31 @@ msgid "race|Neutral"
...
 
 msgid "doubleLined|Wizard"
-msgstr ""
+msgstr "Вълшебник"
```
Comment (Effektus) [inline files/lang/bg.po:6623]: Вълшеб- \n
ник

File: files/lang/bg.po [hunk 9]
```diff
@@ -6608,36 +6608,31 @@ msgid "race|Neutral"
...
 
-#, fuzzy
 msgid ""
 "doubleLined|Necro-\n"
 "mancer"
 msgstr "Некромант"
 
 msgid "doubleLinedRace|Multi"
-msgstr ""
+msgstr "Комбинация"
```
Comment (Effektus) [inline files/lang/bg.po:6631]: Комбина- \n
ция

**Review by Effektus — CHANGES_REQUESTED** [review]
Done

**Comment by Branikolog** [comment]
Hi, @zenseii 

> Hi, @srzkptrsk.
> 
> I only need your input on how to split "Чараўніца", or if you think there is no need for that to be split then please correct me.

I think we can leave them as it is now:
image
Personally, I would like make the word at least 1px thinner, so there would be at least 1 pixel between class names. I believe this could be achieved by future corrections of Cyrillic font.

**Comment by srzkptrsk** [comment]
> Hi, @zenseii
> 
> > Hi, @srzkptrsk.
> > I only need your input on how to split "Чараўніца", or if you think there is no need for that to be split then please correct me.
> 
> I think we can leave them as it is now: image Personally, I would like make the word at least 1px thinner, so there would be at least 1 pixel between class names. I believe this could be achieved by future corrections of Cyrillic font.

Чараў-/nніца will be better.

**Comment by Branikolog** [comment]
> > Hi, @zenseii
> > > Hi, @srzkptrsk.
> > > I only need your input on how to split "Чараўніца", or if you think there is no need for that to be split then please correct me.
> > 
> > 
> > I think we can leave them as it is now: image Personally, I would like make the word at least 1px thinner, so there would be at least 1 pixel between class names. I believe this could be achieved by future corrections of Cyrillic font.
> 
> Чараў-/nніца will be better.

If we make capital letters 1 px thinner, class names would look acceptable:
Thin letters
But anyway, we can return to it later and for now split word into 2 lines, as @srzkptrsk proposed.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
Hi @zenseii , could you please resolve merge conflicts? It's hard for me to do right now as I'm using a phone. Alternatively, I'll do it few hours later.

**Comment by ihhub** [comment]
@zenseii , they appeared after the latest translation PR merge.

**Pull request merged**